examples/fastify/__integration-tests__/fastify.spec.ts

```diff
@@ -300,9 +299,9 @@ data"
 
       await slowFieldResolverInvoked.promise;
       abortController.abort();
-      await expect(response$).rejects.toMatchInlineSnapshot(
```

Comment:
This statement also tests if it throws.
The change below just checks properties of the returned value which can be `Response` object itself if it doesn't throw.

Reply:
Yeah that's technically true. In this case, we care about the client request ending with an abort. Does it matter whether it is thrown? The `instanceof` check ensure the value type is what we expect (not just its to-string'd form as before) and if we're really caring about this coming from promise reject deduction suggests it must be reject unless we believe fetch returned a `DOMException` from the promise success.

But anyways, let's make this airtight with a e.g. `=> ({ __error__: e })` that forces destructing the result from a key that 99.999999% came from the reject. I'll refine now!

Reply:
aside: An `Either` type is far superior and clearer than throwing etc. IMO, but alas, sticking to idiomatic JS here despite what I prefer too :)

Reply:
In this case, if there is no `_`, it will fail `message` of `undefined` and that error would be confusing for the user.
I'd prefer more accurate errors instead of that kind of `TypeError`s;
Maybe something like below works better;
```ts
await expect(
  response$
).rejects.toMatchObject({
   message: 'This operation was aborted',
});
```

Reply:
Yeah nice, like it.